cryptus-app/components/market_viewer/favorite_collections/favorite_collections.module.scss

```diff
@@ -0,0 +1,93 @@
+@use "/styles/variables" as v;
+
+.container {
+  display: flex;
+  flex: 1 1 auto;
+  flex-direction: column;
+  align-items: center;
+  text-align: center;
+
+  width: 100%;
+  position: sticky;
```

Comment:
why does the position need to be sticky? It's a real question I don't know why? Thanks!